terraform/docker-registry.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_ecr_repository" "discourse" {
+  name = "discourse-${terraform.workspace}"
+  tags                        = "${merge(var.common-tags, var.workspace-tags)}"
```

Comment:
Nitpick, formatting here. I recommend running this through `terraform fmt`

Reply:
:+1: I should add a pre commit hook doing this